@teemow as for this error...

```
> make generate
rm -rf client
rm -rf models
rm -rf api-spec/*
# pull spec
mkdir -p ./api-spec
curl -s https://raw.githubusercontent.com/giantswarm/api-spec/master/spec.yaml > api-spec/spec.yaml
/bin/sh: api-spec/spec.yaml: Keine Berechtigung
make: *** [Makefile:9: generate] Fehler 1
```

... I can't explain where the permission error is coming from. Please check out this branch and test if it's working in your environment. Thanks!

--- a/Makefile
+++ b/Makefile
@@ -1,4 +1,6 @@
 PWD := $(shell pwd)
+USERID=$(shell id -u)
+GROUPID=$(shell id -g)
 
 # API Spec branch name
 BRANCH := master
@@ -13,6 +15,7 @@ generate: clean
 	docker run --rm -it \
 	  -v ${PWD}:/go/src/github.com/giantswarm/gsclientgen \
 		-w /go/src/github.com/giantswarm/gsclientgen/api-spec \
+		--user ${USERID}:${GROUPID} \
 		quay.io/goswagger/swagger:0.15.0 \
 			generate client \
 			--spec spec.yaml \
@@ -33,9 +36,11 @@ clean:
 validate:
 	docker run --rm -it \
 	    -v ${PWD}/api-spec:/workdir \
+		--user ${USERID}:${GROUPID} \
 	    boiyaa/yamllint:1.8.1 ./spec.yaml
 	docker run --rm -it \
 		-v ${PWD}/api-spec:/swagger-api/yaml \
+		--user ${USERID}:${GROUPID} \
 		jimschubert/swagger-codegen-cli:2.2.3 generate \
 		--input-spec /swagger-api/yaml/spec.yaml \
 		--lang swagger \

